installer/scripts/service_control

```diff
@@ -93,6 +102,7 @@ start_omsagent_process()
 
 start_omsagent()
 {
+    exit_if_agent_not_onboarded
```

Comment:
Prefer that you verify that the service is enabled before start and restart. So, that is, upon start:

- Register the service if we are onboarded and if service is not yet registered
- Start the service

Verify that restart has similar logic, or doesn't need it if it relies on 'start'.

Enable should still exist, as it makes sense to call that from the post installation scripts. So don't do away with it, just include it if we say "start" and the service isn't enabled.